src/getCartReceptors.C

```diff
@@ -37,15 +37,15 @@ void MeshBlock::getCartReceptors(CartGrid *cg,parallelComm *pc)
   int i,j,k,l,m,c,n,itm,jj,kk;
   int i3;
   int iflag;
-  int icount,dcount;
+  int icount,dcount,cell_count;
   int nsend,nrecv;
   int *pmap;
   int *sndMap,*rcvMap;
   OBB *obcart;
   INTEGERLIST2 *head;
   INTEGERLIST2 *dataPtr;
   double *xtm;
-  double xd[3];
+  double xd[3],vol;
```

Comment:
We are in C++ world, use local scoped variables where it is going to be used.

Reply:
@sayerhs I have addresed this in 72c3a30